packages/web-components/src/components/lightbox-media-viewer/__stories__/lightbox-media-viewer.stories.ts

```diff
@@ -129,6 +129,7 @@ export const WithCarousel = ({ parameters }) => {
       @dds-expressive-modal-beingclosed="${handleBeforeClose}"
       @dds-expressive-modal-closed="${onClose}"
     >
+      <span slot="header" class="bx--visually-hidden">Lightbox modal with carousel</span>
```

Comment:
Minor comment: I think we should note this hidden content for accessibility in the Docs tab. We need to let adopters know that this contextual label should still be accessible to screenreaders while being removed from the interface for sighted users.

Reply:
Addressed in this commit - https://github.com/carbon-design-system/carbon-for-ibm-dotcom/pull/9149/commits/ce744633ee1a7bcf38352fae145b8c2e7c09c5e9